MediaAdminBundle/Transformer/FolderTransformer.php

```diff
@@ -40,6 +64,23 @@ public function transform($folder)
     }
 
     /**
+     * @param FacadeInterface $facade
+     * @param mixed|null      $source
+     *
+     * @return mixed
+     */
+    public function reverseTransform(FacadeInterface $facade, $source = null)
+    {
+        if ($source instanceof MediaFolderInterface) {
+            $parent = $this->folderRepository->findOneById($facade->parentId);
+            $source->setParent($parent);
+            $source->setPath($parent->getPath() . '/' . $source->getFolderId());
```

Comment:
I think it's not the logic : in this transformer, the parent is changed that's right. So the path of the folder must be updated to take this in account. Then as the folder has changed, an event is raised.
One subscriber may listen to it, or not, that's not the problem of this transformer. The folder update can't be delegated to an eventual subscriber, it's the task of this transformer.